README.md

```diff
@@ -63,17 +70,31 @@ fi
 ```
 
 If you don't already have a `.git/hooks/pre-commit` file in your `docs` git repo directory, you can copy the existing `pre-commit.sample` file in that directory as `pre-commit` to use the sample template, or create a new `pre-commit` file in that directory with the above content.
-If you create a new file, you must also make it executable with: `chmod 755 /path/to/my/.git/hooks/pre-commit`.
+If you create a new file, you must also make it executable with: `chmod +x /path/to/my/.git/hooks/pre-commit`.
 
-To ensure your markdown is properly formatted, run `make markdowntest`.
+### Check markdown formatting
+
+To ensure your markdown is properly formatted, run:
+
+```console
+make markdowntest docs/**/*`
+```
+
+### Check for broken links
+
+To check locally for broken links, install [`htmltest`](https://github.com/wjdp/htmltest):
+
+```console
+brew install htmltest
```

Comment:
Nice! However, [opt] could mention this is for mac and/or point non-mac users in the right direction. Majority of employees are using macs but external contributors and a handful of Viam folks have other machines.

Reply:
Very very true. However, most of the rest of the docs don't cater to non-macOS users, and honestly when I used Linux for work I ironically found the brew formula registry (https://formulae.brew.sh/formula/htmltest in this case) one of the best ways to find install steps for Linux, because the pages are short, simple, and always link to the source code and project homepage.

Backlog addition: add info for Linux users throughout the README. I just wish GFM (github-flavored markdown) would add support for tabs, because lining up installation instructions for each platform sequentially is kind of a pain.

Reply:
Interesting. Yeah this readme doesn't have to cater to Linux now, but I'm surprised you say the docs in general don't--most machines run on Linux (e.g. Pi)!